Component/Avatar/avatar.css

```diff
@@ -1 +1,143 @@
-/* TODO: start making */
\ No newline at end of file
+:root{
+  --primary-color:#7c3aed;
```

Comment:
Define colors in your main css file. If you are following this way, you would end up defining these colors in every `.css` file in your project

Reply:
yes sure